internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -340,6 +341,12 @@ func resourceLinuxVirtualMachine() *pluginsdk.Resource {
 				ValidateFunc: computeValidate.VirtualMachineScaleSetID,
 			},
 
+			"vm_agent_platform_updates_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
Is this the API default? iirc the api actually defaults to true

could you please check this because if it doens't match it would be a breaking change